Healthy/Classes/Modules/Splash/OnboardingViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  OnboardingViewController.swift
+//  Healthy
+//
+//  Created by dodo on 15/05/2023.
+//
+
+import UIKit
+
+class OnboardingViewController : UIViewController {
+    
+    let stackView = UIStackView()
+    
+    let imageView = UIImageView()
+    let titleLabel = UILabel()
+    let subtitleLabel = UILabel()
+    
+    init(imageName: String, titleText: String, subtitleText: String) {
+        super.init(nibName: nil, bundle: nil)
+        imageView.image = UIImage(named: imageName)
+
+        titleLabel.text = titleText
+        subtitleLabel.text = subtitleText
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor(patternImage: UIImage(named: "Background")!)
```

Comment:
I recommend using SwiftGen to handle Colors, Images, and Strings
https://github.com/SwiftGen/SwiftGen